starcompactor/formatters/csv.py

```diff
@@ -45,7 +39,7 @@
 
 def csv_row(event, trace_type, instance_type):
     properties = {}
-    for k in event.keys():
+    for k in list(event.keys()):
```

Comment:
I don't think you need `list` here - if you used 2to3, it will be very conservative with this type of thing. Since you're using it like an iterator (`for ... in ...`), casting should not be required.

https://stackoverflow.com/questions/27476079/why-does-2to3-change-mydict-keys-to-listmydict-keys

Reply:
It needs to delete some items from the dict. I can not use "del dict[key]" or ".pop()", since .keys() returns iterator in python3. It will give "RuntimeError: dictionary changed size during iteration".

Reply:
Gotcha, that's definitely valid, and I missed that you were modifying the dict.